tunnel.go

```diff
@@ -13,19 +13,25 @@ import (
 
 // Tunnel is the struct definition of a tunnel.
 type Tunnel struct {
-	ID          string             `json:"id,omitempty"`
-	Name        string             `json:"name,omitempty"`
-	Secret      string             `json:"tunnel_secret,omitempty"`
-	CreatedAt   *time.Time         `json:"created_at,omitempty"`
-	DeletedAt   *time.Time         `json:"deleted_at,omitempty"`
-	Connections []TunnelConnection `json:"connections,omitempty"`
+	ID             string             `json:"id,omitempty"`
+	Name           string             `json:"name,omitempty"`
+	Secret         string             `json:"tunnel_secret,omitempty"`
```

Comment:
When will this be used/present?

Reply:
when creating a new tunnel and the response will include the secret (the only time).

Reply:
it's probably less of an issue now that you can fetch the token via the API though.